src_assets/common/assets/web/config.html

```diff
@@ -502,6 +502,21 @@ <h1 class="my-4">Configuration</h1>
           Disable if you encounter any VSync-related issues.
         </div>
       </div>
+      <!--fps_min-->
+      <div class="mb-3">
+        <label for="qp" class="form-label">FPS Minimum</label>
+        <input
+          type="number"
+          class="form-control"
+          id="fps_min"
+          placeholder="10"
+          v-model="config.min_fps"
+        />
+        <div class="form-text">
+          The minimum framerate Sunshine will attempt to maintain.<br />
+          Increasing this value slightly may help when streaming mostly static content.<br />
+          Higher values will consume more bandwidth.<br />
+        </div>
```

Comment:
You're missing a final closing divider (`</div>`), which is breaking all other tabs/elements after this new setting.